autoload/elixir/indent.vim

```diff
@@ -14,6 +14,7 @@ let s:MULTILINE_BLOCK = '\%(\<do\>'.s:NO_COLON_AFTER.'\|'.s:MULTILINE_FN.'\)'
 let s:BLOCK_MIDDLE = '\<\%(else\|match\|elsif\|catch\|after\|rescue\)\>'
 let s:BLOCK_END = 'end'
 let s:STARTS_WITH_PIPELINE = '^\s*|>.*$'
+let s:QUERY_FROM = '^\s*from\s*.*\s*in\s*.*,'
```

Comment:
I think we need to improve this regexp, today it matchs with undesired texts too:
<img width="286" alt="nvim" src="https://cloud.githubusercontent.com/assets/120483/21651920/789e8434-d2a2-11e6-964b-c3c9ea1da4e7.png">

I'd use something like: `^\s*\<from\>.*\<in\>.*,`, wdyt?